readers-frontend/src/pages/index.astro

```diff
@@ -44,3 +44,42 @@ const categories = await getFeaturedSectionsCategories();
     </div>
   </main>
 </Layout>
+
+<script define:vars={{ categories }}>
```

Comment:
I didn't want to move it to a general layout because this script is only relevant on the Nouse and Muse homepages. Also, the script has to change the colour of the buttons and logo on the Nouse side. Because of that I didn't put it in a component.

Reply:
Could still be a separate component with an `isNouse` prop to enable the Nouse behaviour no?

Reply:
I mean it could be. It's just whether we're actually omitting much code redundancy that's all

Reply:
Yeah, why would we keep the same static javascript in two locations? I haven't tried it but this looks interesting https://docs.astro.build/en/guides/client-side-scripts/#pass-frontmatter-variables-to-scripts So the script would still be bundled